server/src/main/java/dev/slimevr/serial/SerialHandler.java

```diff
@@ -232,7 +281,9 @@ public int hash(SerialPort o) {
 			lastKnownPorts = SerialPort.getCommPorts();
 			differences.forEach(this::onNewDevice);
 		} catch (Throwable e) {
-			LogManager.severe("Using serial ports is not supported on this platform", e);
+			LogManager
+				.severe("[SerialHandler] Using serial ports is not supported on this platform", e);
+			throw new RuntimeException("Serial unsupported");
```

Comment:
Why are we getting rid of the original exception? I don't think it's safe to do this, as it crashes the server when it's unsupported. If this is being caught somewhere else, can I be pointed to it?

Reply:
Yes, the following tasks scheduled in the timer wont run because background thread is dead